README.md

```diff
@@ -23,31 +23,65 @@ There are three modules this project:
       It instruments the java.util.logging.Logger.log using JavaAssist.
 
 4. riddles folder \
-    The riddles folder contains two riddles that can be solved using the profiler. 
+    The riddles folder contains two riddles that can be solved using the profiler you will write. 
     You can find there a `README` file with the run instructions. 
 
-Prepare work environment with Maven and IDE 
+Prepare work environment with GitHub Codespaces.
 --------------------------------------------
-To prepare your work environment, follow these steps:
+
+## 🧑‍💻 Option 1: Use VS Code in the Browser
+
+1. Go to the GitHub repository: [JavaInstrumentationWorkshop](https://github.com/irachle/JavaInstrumentationWorkshop)
+2. Click the green **Code** button.
+3. Select **Codespaces → Create codespace on main**.
+4. Wait for the environment to initialize. It will automatically:
+    - Clone the workshop repo
+    - Install Java 1.8, Maven, and Git
+    - Set up Java extensions for VS Code
+
+You can now start coding directly in your browser!
+
+---
+
+## 🧠 Option 2: Use IntelliJ IDEA via JetBrains Gateway
+
+1. Download and install [JetBrains Gateway](https://www.jetbrains.com/remote-development/gateway/)
+2. Open JetBrains Gateway and install the **GitHub Codespaces plugin**

```

Comment:
can we get a direct link, or more detailed instructions? Chances are that the participants aren't familiar with JetBrains Gateway.